tsconfig.json

```diff
@@ -18,7 +18,7 @@
     "isolatedModules": true,
     "downlevelIteration": true
   },
-  "exclude": ["node_modules", "**/dist/**"],
+  "exclude": ["node_modules", "**/dist/**", "fixtures/next-app-router/**/*"],
```

Comment:
oops, I wrote some comments on my fork and forgot to add them here.
There was some type conflicts happening where react router in the site package was picking up react 18 types from the `next-app-router` package. 
Had another look and I think this can be resolved by not using react 18 types at all.